.github/actions/version-matrix/src/matrices/node/playwright.ts

```diff
@@ -0,0 +1,56 @@
+import { supportedNodeVersions } from '../../shared/constants';
+import { fetchPackageVersions } from '../../shared/npm';
+
+const playwrightVersions = await fetchPackageVersions('playwright');
+const apifyVersions = await fetchPackageVersions('apify');
+const crawleeVersions = await fetchPackageVersions('crawlee');
+
+const shouldUseLastFive = process.env.GITHUB_EVENT_NAME ? process.env.GITHUB_EVENT_NAME !== 'pull_request' : true;
```

Comment:
Could we define a `SHOULD_USE_LAST_FIVE` environment variable at the workflow level instead, so that stuff like event names doesn't seep into the scripts?